src/Model/Scope/BasicCondition.php

```diff
@@ -265,7 +265,7 @@ protected function keyToWords(bool $asHtml = false): string
 
     protected function operatorToWords(bool $asHtml = false): string
     {
-        return $this->operator ? (self::$dictionary[strtoupper((string) $this->operator)] ?? 'is equal to') : '';
+        return $this->operator ? (self::$dictionary[strtoupper((string) $this->operator)] ?? $this->operator) : '';
```

Comment:
why? - it should emit at least php notice, you already test if operator is not empty in the first condition

if operator can be other than supported/listed, this is correct, but do we support (and want to support it) that everywhere?

Reply:
Changed